samples/openvx/openvx_interoperability.cpp

```diff
@@ -0,0 +1,324 @@
+#include <iostream>
+#include <stdexcept>
+
+//OpenVX includes
+#include <VX/vx.h>
+#include <VX/vx_lib_debug.h>
+#include <VX/vx_helper.h>
+#include <VX/vx_lib_extras.h>
+
+//for vx-Access/Commit-ImagePatch
+#include <VX/vx_compatibility.h>
+
+//OpenCV includes
+//#include <opencv2/opencv.hpp>
+#include "opencv2/core.hpp"
+#include "opencv2/imgproc.hpp"
+#include "opencv2/imgcodecs.hpp"
+#include "opencv2/highgui.hpp"
+
+// vx_debug.h
+extern "C" void vx_set_debug_zone(vx_enum zone);
```

Comment:
Again, this is an artifact of sample implementation. In sample implementation this debug zones can be controlled by environment variables.